### PR TITLE
Ansible: Install systemd service from official tarball, rather than source

### PR DESCRIPTION
**Pull request by moreati**
Deploy nodes using official binaries, rather than building from source.This

- speeds up node upgrades
- eliminates need for a build environment on a node
- reduces load on a running node during upgrade
- ensures validated, tested binaries are used

This only affects service_playbook.yml. Other playbooks will continue to build from source.

### CODE REVIEW

**Comment by StephenButtolph** [comment]
Thanks as always for this - are you aware of https://github.com/ava-labs/avalanchego#binary-repository? I feel like using an APT repo may be the easiest way to mange installs

**Comment by moreati** [comment]
> Thanks as always for this - are you aware of https://github.com/ava-labs/avalanchego#binary-repository?

I am, but I found I couldn't use binaries in the Debian packages failed to run the evm plugin. I was going to investigate further later, I'll raise an issue with logs etc, once I reproduce the error.

**Comment by StephenButtolph** [comment]
Would you mind re-basing this off of `dev`? The avalanchego daemon was refactored for the next release.

**Comment by moreati** [comment]
> Would you mind re-basing this off of dev?

That should do it, sorry for the rebase notifications noise.

**Review by aaronbuchwald — DISMISSED** [review]
LGTM

File: scripts/ansible/README.md [hunk 1]
```diff
@@ -127,76 +127,80 @@ Target operating systems supported by these roles & playbooks are
...
 changed: [localhost] => (item=/var/local/log/avalanchego)
 changed: [localhost] => (item=/usr/local/lib/avalanchego)
 
-TASK [avalanche_install : Install Avalanche binary] ***********************************
+TASK [avalanche_install : Install Avalanche binary] ****************************
 changed: [localhost]
 
-TASK [avalanche_install : Install Avalanche plugins] **********************************
-changed: [localhost] => (item={'path': '~auser/go/src/github.com/ava-labs/avalanchego/build/plugins/evm'})
+TASK [avalanche_install : Remove outdataed support files] **********************
```
Comment (StephenButtolph) [inline scripts/ansible/README.md:178]: super minor nit (that isn't really related to this PR) but I just noticed that this seems to be a typo.

outdataed -> outdated

**Review by StephenButtolph — COMMENTED** [review]
after this LGTM

File: scripts/ansible/README.md (hunk 1 above)
Comment (moreati) [reply scripts/ansible/README.md:178]: ty, I'll do a sweep for others as well

**Review by StephenButtolph — APPROVED** [review]

**Pull request merged**